pkg/deploy/controller/deploymentconfig/controller.go

```diff
@@ -267,8 +278,7 @@ func (c *DeploymentConfigController) findDetails(config *deployapi.DeploymentCon
 		multipleErrWarning := fmt.Sprintf("Deployment config %q blocked by multiple errors:\n", config.Name)
 		allDetails = append([]string{multipleErrWarning}, allDetails...)
 	}
-
-	return strings.Join(allDetails, "\n"), existingDeployments, latestDeploymentExists, nil
+	return strings.Join(allDetails, "\n"), nil
```

Comment:
We may want to change all `strings` calls with their `bytes` respective. eg. https://golang.org/pkg/bytes/#Join